prov/sm2/src/sm2_ep.c

```diff
@@ -240,20 +240,18 @@ sm2_gid_t sm2_verify_peer(struct sm2_ep *ep, fi_addr_t fi_addr, sm2_gid_t *gid)
 	return 0;
 }
 
-void sm2_generic_format(struct sm2_xfer_entry *xfer_entry, sm2_gid_t self_gid,
-			uint32_t op, uint64_t tag, uint64_t cq_data,
-			uint64_t op_flags)
+static void sm2_generic_format(struct sm2_xfer_entry *xfer_entry,
+			       sm2_gid_t self_gid, uint32_t op, uint64_t tag,
+			       uint64_t cq_data, uint64_t op_flags,
+			       void *context)
```

Comment:
why does clang realign this function when a parameter was just added to the end? it makes the changes hard to read

Reply:
I added static keyword.